README.md

```diff
@@ -1,7 +1,86 @@
 # hocs-toolbox
 
-This image contains the abilities to access the namespace database, and also provides scripts for clearing the database, Elasticsearch, S3 buckets and SQS queues.
+This branch is customised for the CMS data migration, it is a toolbox with Microsoft sqlcmd and the AWS CLI installed. It's main purpose is to allow the team to copy the CMS database backup file to an S3 bucket and then restore the database to the SQL Server 2016 database in the cluster.
 
+To Access the deployment and work on SQL Server via CLI, please follow the steps
 
 
+https://hub.acp.homeoffice.gov.uk/identities
 
+## Set up Kube config to access acp-notprod cluster
+1. Set up the cluster settings:
+```
+kubectl config set-cluster acp-notprod --server=https://kube-api-notprod.notprod.acp.homeoffice.gov.uk && kubectl config set clusters.acp-notprod.certificate-authority-data xxxxxxxxx
+```
+
+2. Set up the credentials:
+
+```
+kubectl config set-credentials user@digital.homeoffice.gov.uk_acp-notprod_HOCS --token=xxxxxx
+```
+
+3. Set up the context:
+
+```
+kubectl config set-context acp-notprod_HOCS --cluster=acp-notprod --user=user@digital.homeoffice.gov.uk_acp-notprod_HOCS
+```
+
+4. Now you can access the cluster: to list the pods in the namespace
+
+```
+kubectl --context=acp-notprod_HOCS --namespace=cs-dev-migration get pods
+```
+
+Displays the hooks-migration-toolbox
+
+NAME                                      READY   STATUS    RESTARTS   AGE
+hocs-migration-toolbox-76485cf569-7xhdn   1/1     Running   0          94m
+
+We can connect to this container by
+
+```
+kubectl --context=acp-notprod_HOCS --namespace=cs-dev-migration exec -it deployment/hocs-migration-toolbox -- bash
+```
+
+Once we are in the container, we can connect to the Sql Server using sqlCmd
+
+```
+sqlcmd -S $CMS_DB_HOSTNAME -U $CMS_DB_USERNAME -P $CMS_DB_PASSWORD
```

Comment:
You might want to make this into a Bash script (see the others in scripts/safe/db) to make this easier to launch